lib/modules/manager/gomod/artifacts.spec.ts

```diff
@@ -1580,4 +1580,42 @@ describe('modules/manager/gomod/artifacts', () => {
     ];
     expect(execSnapshots).toMatchObject(expectedResult);
   });
+
+  it('handles goGetDirs configuration correctly', async () => {
+    fs.readLocalFile.mockResolvedValueOnce('Current go.sum');
+    fs.readLocalFile.mockResolvedValueOnce(null); // vendor modules filename
+    const execSnapshots = mockExecAll();
+    git.getRepoStatus.mockResolvedValueOnce({
+      modified: [] as string[],
+    } as StatusResult);
```

Comment:
use partial function from test utils

Reply:
sure, I updated to use `partial`, but all the other tests in the same file don't use it :shrug: